### ISSUE TITLE
Siri no longer but homekit app does

### ISSUE THREAD

**Issue by leonmf** [nfarina/homebridge#731]
It seems that my Eve homekit app still works to control everything, including devices. However, Siri claims to not be able to communicate with devices. Some (maybe all) scenes still seem to work. Has Apple gone and changed something or is there something wrong with my system?

Z-Way running on rPi2
Homebridge running on rPi3
ios 9.3.3
Eve homekit app.

**Comment by NewFolk** [comment]
Same too me. it`s look like siri does not understand rooms name on Russian. Like Kitchen - "Кухня"
Domoticz + https://github.com/PatchworkBoy/homebridge-eDomoticz 
iOS 9.3.4 
rPi3
Eve + Insteon app work Ok

**Comment by jaysettle** [comment]
I had this same issue for about 2 days. It happened after I shared iCloud HomeKit with my daughters iPad. I invited her iCloud email through one of the apps that work it HomeKit and I don't know if it was immediate but I remember her iPad's Siri would still work but mine did not. I ended up making a new home and restarting the server. Can't remember, but I might have turned iCloud off and back on on my iPhone.  


-Jay
